api/entrypoint.sh

```diff
@@ -4,7 +4,7 @@
 host=db; port=3306; n=120; i=0; while ! (echo > /dev/tcp/$host/$port) 2> /dev/null; do [[ $i -eq $n ]] && >&2 echo "$host:$port not up after $n seconds, exiting" && exit 1; echo "waiting for $host:$port to come up"; sleep 1; i=$((i+1)); done
 
 # collect static files
-python manage.py collectstatic --clear --noinput || exit 1
```

Comment:
The `collectstatic` command is designed to move static files from their locations in the different Django modules into a common directory. Afaik, it is not best practice to include static files into the common directory yourself. Please move the static files into their [canonical location](https://docs.djangoproject.com/en/1.11/ref/contrib/staticfiles/).

Also, keep in mind that you're adding duplicate code here. Much of the static files added here was already added in earlier commits (cf #25 )